geopandas/array.py

```diff
@@ -67,31 +72,42 @@ def _isna(value):
 # -----------------------------------------------------------------------------
 
 
+def _geom_to_shapely(geom):
+    if not compat.USE_PYGEOS:
+        return geom
+    if geom is None:
+        return None
+    elif pygeos.is_empty(geom) and pygeos.get_type_id(geom) == 0:
```

Comment:
Can we add a comment with ref to that shapely issue to make clear why is this separately?